test/test_sparse.py

```diff
@@ -3251,26 +3251,49 @@ class TestSparseUnaryUfuncs(TestCase):
 
     @ops(sparse_unary_ufuncs)
```

Comment:
Add a TODO: that this test will need to be extended for non-unary operations

Reply:
Put in a TODO as suggested